recipes/secretstorage/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "SecretStorage" %}
+{% set version = "3.0.1" %}
+{% set sha256 = "819087ca89c0d6c5711692f41fb26f786af9dcc5bb89d567722a66edfbb2a689" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [win or osx or py<35]
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - jeepney
+    - cryptography
+
+test:
+  source_files:
+    - tests
+  imports:
+    - secretstorage
+  commands:
+    - python3 -m unittest discover -s tests 
```

Comment:
Try this instead
`dbus-run-session -- python3 -m unittest discover -s tests`

Reply:
`ine 7: dbus-run-session: command not found`